lwt/happy_eyeballs_lwt.mli

```diff
@@ -26,3 +32,5 @@ val connect : t -> string -> int list ->
     may be a host name, or an IP address.
 
     @raise Failure if [ports] is the empty list. *)
+
+val inject : getaddrinfo:getaddrinfo -> t -> unit
```

Comment:
there could be a docstring. Also, the label for the first argument isn't necessary.